src/modules/render.js

```diff
@@ -58,7 +55,11 @@ const showPopup = async (ebook) => {
   commentsArray.sort((a, b) => new Date(b.creation_date) - new Date(a.creation_date));
 
   // displaying comments
-  const allComments = `<div class="comment-list">${showComments(commentsArray)}</div>`;
+  let allComments = '<div class="comment-list">';
+  commentsArray.forEach((x) => {
+    allComments += showComments(x);
+  });
```

Comment:
We can improve ES6 by writing this forEach in one line only. And change the "x" named variable to another more meaningful one.

Reply:
Hi @lmunozm1702, writing function into one line is throwing linter error "Arrow function should not return assignment  no-return-assign". I just went ahead with renaming the variable.